mock-relying-party-ui/README.md

```
@@ -18,52 +18,37 @@ This portal contains 2 pages:
+| **DISPLAY**                       | Specifies how the authorization server displays the authentication and consent screen             | `page`, `popup`, `wap`, `touch` (e.g., `page`)           | Optional                                                                                                                                                                 |
+| **PROMPT**                        | Specifies the type of prompt to show during the authentication flow                               | `consent`                                                | Optional                                                                                                                                                                 |
+| **PAR_CALLBACK_NAME**             | **Feature flag** to enable PAR (Pushed Authorization Request) flow                                | `get_requestUri`                                         | Hardcoded function name - not configurable                                                                                                                               |
+| **PAR_CALLBACK_TIMEOUT**          | Timeout for PAR callback in milliseconds                                                          | `5000`                                                   | Optional (Default: `5000` / 5s)                                                                                                                                          |
+| **DPOP_CALLBACK_NAME**            | **Feature flag** to enable DPoP (Demonstration of Proof-of-Possession) flow                       | `get_dpop_jkt`                                           | Hardcoded function name - not configurable                                                                                                                               |
+| **CODE_CHALLENGE**                | **Feature flag** to enable PKCE (Proof Key for Code Exchange) flow                                | `get_code_challenge`                                     | Hardcoded function name - not configurable.<br><br>When enabled, the PKCE method is automatically fetched from the server's `.well-known/openid-configuration` endpoint. |
 
 - Build and run Docker for a service:
 
-  ```
+  ```bash
   $ docker build -t <dockerImageName>:<tag> .
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)